docs/configuration/config-options.md

```diff
@@ -209,7 +209,7 @@ variables are passed to the `options` configuration of a
 [`Pino-http` instance](https://github.com/pinojs/pino-http#api). Based on your project's needs, you can extend the
 `LOGGER_*` environment variables with any config you need to pass to the logger instance. If a LOGGER_LEVELS key is
 added, these values will be passed to the logger formatter, as described
-[here](https://github.com/pinojs/pino/blob/master/docs/help.md#mapping-pino-log-levels-to-google-cloud-logging-stackdriver-serverity-levels)
+[here](https://github.com/pinojs/pino/blob/master/docs/help.md#mapping-pino-log-levels-to-google-cloud-logging-stackdriver-severity-levels)
```

Comment:
This is currently misspelled, but I expect to have a PR for them shortly.

Reply:
https://github.com/pinojs/pino/commit/174723eabcc05b8e4ceccd2bc4d114014719afb2#diff-7bed7f9ea19fe4d3abd5614c8a15877c0148d79a8b7cd1275e549e0f65589d17R204